packages/code-du-travail-frontend/src/outils/common/Agreement/tracking/pushAgreementEvents.ts

```diff
@@ -52,7 +52,7 @@ const pushAgreementEvents = (
         ? MatomoSearchAgreementCategory.AGREEMENT_SELECT_P1
         : MatomoSearchAgreementCategory.AGREEMENT_SELECT_P2,
       title,
-      `idcc${values.selected.num.toString()}`,
+      `idcc${values.selected.num?.toString()}`,
```

Comment:
ça c'est une erreur que j'ai vu dans Sentry ce matin : https://sentry.fabrique.social.gouv.fr/organizations/incubateur/issues/42573/?environment=production&project=5&query=is%3Aunresolved&statsPeriod=14d